react/src/components/organisms/TeaserListing/index.js

```diff
@@ -0,0 +1,169 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+import Collapse from '../../animations/Collapse';
+import SidebarHeading from '../../atoms/headings/SidebarHeading';
+import CompHeading from '../../atoms/headings/CompHeading';
+import Paragraph from '../../atoms/text/Paragraph';
+import Link from '../../molecules/Link';
+import GeneralTeaser from '../GeneralTeaser';
+
+import './style.css';
+
+class TeaserListing extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      open: false
+    };
+  };
+   handleClick = (e) => {
+    e.preventDefault();
+    this.setState((currentState) => ({
+      open: !currentState.open
+    }));
+  };
+
+  render() {
+    const teaser = this.props;
+    const featuredClasses = classNames({
+      'ma__teaser-listing__featured-items': true,
+      'stacked': teaser.stacked,
+      'side-by-side': !teaser.stacked
+    });
+    const columnCount = (teaser.contained && teaser.gridTwoColumns) ? 2 : 3;
+    const itemsClasses = classNames({
+      'ma__teaser-listing__items': true,
+      'ma__teaser-listing__2-col-grid': columnCount === 2,
+      'ma__teaser-listing__3-col-grid': columnCount === 3
+    });
+    const shownNumber = teaser.shownItems || null;
+    const shownItems = shownNumber ? teaser.items.slice(0, shownNumber) : null;
+    const invisibleItems = (shownNumber) ? teaser.items.slice(shownNumber) : [];
+
+    let teaserHeading = 2;
+    teaserHeading = parseInt((teaser.compHeading && teaser.compHeading.level) ? teaser.compHeading.level : teaserHeading) + 1;
+    teaserHeading = parseInt((teaser.sidebarHeading && teaser.sidebarHeading.level) ? teaser.sidebarHeading.level : teaserHeading) + 1;
+
+    return (
+      <section className="ma__teaser-listing">
+        <div className="ma__teaser-listing__container">
+          {(teaser.compHeading) && (<CompHeading {...teaser.compHeading} />)}
+          {(teaser.sidebarHeading) && (<SidebarHeading {...teaser.sidebarHeading} />)}
+          {(teaser.description) && (
+            <div className="ma__teaser-listing__description">
+              <Paragraph { ...teaser.description} />
+            </div>
+          )}
+          {(teaser.featuredItems) && (
+            <div className={featuredClasses}>
+              {teaser.featuredItems.map((teaser, index) => {
+                const key = `featured-listing--item-${index}`;
+                const teaserProps = {
+                  key,
+                  level: teaserHeading,
+                  ...teaser
+                };
+                return<GeneralTeaser {...teaserProps} />;
+              })}
+            </div>
+          )}
+          {(invisibleItems.length > 0) && (
+            <div>
```

Comment:
```suggestion
            <React.Fragment>
```
This fixes the border bug.

Reply:
Updated.